xml/favicons.xml

```diff
@@ -1330,9 +1330,9 @@
     <param pos="0" name="service.vendor" value="pfSense"/>
     <param pos="0" name="service.product" value="pfSense"/>
     <param pos="0" name="service.device" value="Firewall"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:pfsense:pfsense:-"/>
     <param pos="0" name="service.component.vendor" value="nginx"/>
     <param pos="0" name="service.component.product" value="nginx"/>
-    <param pos="0" name="service.component.cpe23" value="cpe:/a:nginx:nginx:-"/>
```

Comment:
Note: it doesn't appear that we're generating CPEs for service.component. This might be an easy addition.